### DISCUSSION TITLE
Platform Support for "Lumary"

### DISCUSSION THREAD

**Discussion by nrtkin** [milo526/homebridge-tuya-web#275] [General]
Hi,

i have some led spots from Lumary wich is the same as smart life products.
I can also control the LED Spot with this plugin some Colors are not the right one but this i think must be done with overruling.

Can you add the platform Support for Lumary products please?

Here is the log
`
[TuyaWebPlatform] Created new Accessory [Name: Lumary smart slim downlight] [Tuya ID: 0000] [HomeBridge ID: 00000]
[TuyaWebPlatform] [Lumary smart slim downlight] Characteristic.On - {"format":"bool","perms":["ev","pr","pw"]}
[TuyaWebPlatform] [Lumary smart slim downlight] Characteristic.Brightness - {"format":"int","perms":["ev","pr","pw"],"unit":"percentage","minValue":0,"maxValue":100,"minStep":1}
[TuyaWebPlatform] [Lumary smart slim downlight] Characteristic.ColorTemperature - {"format":"int","perms":["ev","pr","pw"],"minValue":140,"maxValue":500,"minStep":1}
[TuyaWebPlatform] [Lumary smart slim downlight] Characteristic.Hue - {"format":"float","perms":["ev","pr","pw"],"unit":"arcdegrees","minValue":0,"maxValue":360,"minStep":1}
[TuyaWebPlatform] [Lumary smart slim downlight] Characteristic.Saturation - {"format":"float","perms":["ev","pr","pw"],"unit":"percentage","minValue":0,"maxValue":100,"minStep":1}

[TuyaWebPlatform] [Lumary smart slim downlight] - Characteristic Name not supported
`

**Comment by nrtkin** [comment]
One more question, how can i switch in HOOBS from color light to withe color light?

Thank you

**Comment by lothsun** [comment]
Would love to see lumary support as well.
